cdap-ui/app/cdap/components/DataPrep/DataPrepBrowser/DatabaseBrowser/index.js

```diff
@@ -111,7 +111,22 @@ export default class DatabaseBrowser extends Component {
           window.location.href = `${window.location.origin}/cdap/ns/${namespace}/dataprep/${workspaceId}`;
         },
         (err) => {
-          setError(err);
+          let error = err;
+          let errorMessage = T.translate(`${PREFIX}.defaultErrorMessage`, {tableId});
+          if (
+            !err ||
+            isNil(err.response) ||
```

Comment:
since line 119 already using objectQuery, do we need to check `err.response` as well?

Reply:
👍 The condition logic was incorrect. Changed it to handle empty err responses to show appropriate error message.